sklearn/utils/_testing.py

```diff
@@ -747,99 +750,166 @@ def _convert_container(
     ----------
     container : array-like
         The container to convert.
-    constructor_name : {"list", "tuple", "array", "sparse", "dataframe", \
-            "series", "index", "slice", "sparse_csr", "sparse_csc", \
-            "sparse_csr_array", "sparse_csc_array", "pyarrow", "polars", \
-            "polars_series"}
+
+        - Converting to "dataframe" requires 2D container
+        - Converting to "series" and "index" requires 1D container
+
+    constructor_type : {"list", "tuple", "slice", "array", "dataframe", "series", \
+                        "index"}
         The type of the returned container.
-    columns_name : index or array-like, default=None
-        For pandas container supporting `columns_names`, it will affect
-        specific names.
+
     dtype : dtype, default=None
-        Force the dtype of the container. Does not apply to `"slice"`
-        container.
+        Force the dtype of the container. Does not apply to "slice".
+
+    sparse_container : {"matrix", "array"}, default=None
+        The sparse container to use. Only applies to "array". Note that if this
```

Comment:
I don't understand ``Only applies to "array"``.  array here refers to the constructor type ? Maybe
```suggestion
        The sparse container to use. Only applies when `constructor_type="array"`. Note that if this
```